src/Query/Grammar.php

```diff
@@ -10,7 +10,6 @@ class Grammar extends MySqlGrammar
     /**
      * Compile a "where fulltext" clause.
      *
-     * @param  \Illuminate\Database\Query\Builder  $query
```

Comment:
why are these doc strings deleted?

Reply:
It is made automatically by a linter after I pushed the code